### DISCUSSION TITLE
Supported typescript?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4728] [Q&A] [answered]
greetings.

I'm trying to use PocketBase in TypeScript.
However, the following error is displayed:
If you have a solution to solve this, please let me know.

![スクリーンショット 2024-04-11 19 24 00](https://github.com/pocketbase/pocketbase/assets/51219166/5bea66d1-97ad-40c4-b0bb-d5a17cc36015)

**Comment by ganigeorgiev** [comment, marked as answer]
There is not enough information to understand or reproduce the issue.

The JS SDK should work fine with a TypeScript based node setup. Double check your TS configuration and make sure that you are loading the JS SDK as ESM (aka. `import ...`) and not CJS (`require(...)`).
